orchestrator/crates/prover-clients/atlantic-service/tests/lib.rs

```diff
@@ -62,10 +64,86 @@ async fn atlantic_client_get_task_status_works() {
         atlantic_verifier_contract_address: get_env_var_or_panic(
             "MADARA_ORCHESTRATOR_ATLANTIC_VERIFIER_CONTRACT_ADDRESS",
         ),
+        atlantic_network: get_env_var_or_panic("MADARA_ORCHESTRATOR_ATLANTIC_NETWORK"),
     };
-    let atlantic_service = AtlanticProverService::new_with_args(&atlantic_params);
+    let atlantic_service = AtlanticProverService::new_with_args(&atlantic_params, &LayoutName::dynamic);
 
-    let atlantic_query_id = "01JDY6EKVQD8QYR8HE64WZC9VB";
+    let atlantic_query_id = "01JPMKV7WFP4JTC0TTQSEAM9GW";
     let task_result = atlantic_service.atlantic_client.get_job_status(atlantic_query_id).await;
     assert!(task_result.is_ok());
 }
+
+#[tokio::test]
+async fn atlantic_client_submit_task_and_get_job_status_with_mock_fact_hash() {
+    let _ = env_logger::try_init();
+    dotenvy::from_filename("../.env.test").expect("Failed to load the .env file");
+
+    // Initialize Atlantic parameters from environment variables
+    let atlantic_params = AtlanticValidatedArgs {
+        atlantic_api_key: get_env_var_or_panic("MADARA_ORCHESTRATOR_ATLANTIC_API_KEY"),
+        atlantic_service_url: Url::parse(&get_env_var_or_panic("MADARA_ORCHESTRATOR_ATLANTIC_SERVICE_URL")).unwrap(),
+        atlantic_rpc_node_url: Url::parse(&get_env_var_or_panic("MADARA_ORCHESTRATOR_ATLANTIC_RPC_NODE_URL")).unwrap(),
+        atlantic_mock_fact_hash: get_env_var_or_panic("MADARA_ORCHESTRATOR_ATLANTIC_MOCK_FACT_HASH"),
+        atlantic_prover_type: get_env_var_or_panic("MADARA_ORCHESTRATOR_ATLANTIC_PROVER_TYPE"),
+        atlantic_settlement_layer: get_env_var_or_panic("MADARA_ORCHESTRATOR_ATLANTIC_SETTLEMENT_LAYER"),
+        atlantic_verifier_contract_address: get_env_var_or_panic(
+            "MADARA_ORCHESTRATOR_ATLANTIC_VERIFIER_CONTRACT_ADDRESS",
+        ),
+        atlantic_network: get_env_var_or_panic("MADARA_ORCHESTRATOR_ATLANTIC_NETWORK"),
+    };
+
+    // Create the Atlantic service with actual configuration
+    let atlantic_service = AtlanticProverService::new_with_args(&atlantic_params, &LayoutName::dynamic);
+
+    // Load the Cairo PIE from the test data
+    let cairo_pie_path = env!("CARGO_MANIFEST_DIR").to_string() + CAIRO_PIE_PATH;
+    print!("Loading Cairo PIE from {}", cairo_pie_path);
+    let cairo_pie = CairoPie::read_zip_file(cairo_pie_path.as_ref()).expect("Failed to read Cairo PIE zip file");
+
+    // Submit the task to the actual Atlantic service
+    let task_result = atlantic_service
+        // We don't need to send the steps because it's a mock fact hash.
+        .submit_task(Task::CairoPie(Box::new(cairo_pie)), None)
+        .await
+        .expect("Failed to submit task to Atlantic service");
```

Comment:
```suggestion
     { 
           let cairo_pie = CairoPie::read_zip_file(cairo_pie_path.as_ref()).expect("Failed to read Cairo PIE zip file");

    // Submit the task to the actual Atlantic service
    let task_result = atlantic_service
        // We don't need to send the steps because it's a mock fact hash.
        .submit_task(Task::CairoPie(Box::new(cairo_pie)), None)
        .await
        .expect("Failed to submit task to Atlantic service");
 }
```

we can drop cairoPIE to make sure it's not clogging our machine's memory (same logic as you applied in the snos jobs?) wdyt?

Reply:
I understand your concerns, but I wanna confirm wether do we really want to do it in `tests` ?